src/main/java/com/gd/clinic/enums/DaysEnum.java

```diff
@@ -0,0 +1,44 @@
+package com.gd.clinic.enums;
+
+//TODO: Implement enums from remaining entities/DTOs
+public enum DaysEnum {
+
+    MONDAY("Monday"),
+
+    TUESDAY("Tuesday"),
+
+    WEDNESDAY("Wednesday"),
+
+    THURSDAY("Thursday"),
+
+    FRIDAY("Friday"),
+
+    SATURDAY("Saturday"),
+
+    SUNDAY("Sunday");
+
+    private String value;
+
+    DaysEnum(String value) {
+        this.value = value;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    @Override
+    public String toString() {
+        return String.valueOf(value);
+    }
+
+    public static DaysEnum fromValue(String value) {
+        for (DaysEnum b : DaysEnum.values()) {
```

Comment:
this can also be refactored like this 
```
Arrays
            .stream(DaysEnum.values())
            .filter(daysEnum -> daysEnum.value.equals(value))
            .findFirst()
            .orElseThrow(() -> you exception);
```